src/zenml/zen_service/zen_service.py

```diff
@@ -0,0 +1,122 @@
+from pathlib import Path
+from typing import Dict, List
+
+from fastapi import FastAPI
+
+from zenml.enums import StackComponentType
+from zenml.io.utils import get_global_config_directory
+from zenml.stack_stores import BaseStackStore, SqlStackStore
+from zenml.stack_stores.models import (
+    ActiveStackName,
+    StackComponentConfiguration,
+    StackConfiguration,
+    StackWrapper,
+    Version,
+)
+
+app = FastAPI()
+
+# to run this, execute:
+# uvicorn zenml.zen_service.zen_service:app --reload
+
+
+root: Path = Path(get_global_config_directory())
+url = f"sqlite:///{root / 'service_stack_store.db'}"
+print(url)
```

Comment:
Maybe some explanation what the URL means? I think also some docstrings in this file would probably be nice to show up in the API-docs, even if it's just a one-liner each